src/Utils/Numbers.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/**
+ * This file is part of the Nette Framework (https://nette.org)
+ * Copyright (c) 2004 David Grudl (https://davidgrudl.com)
+ */
+
+declare(strict_types=1);
+
+namespace Nette\Utils;
+
+use Nette;
+
+
+/**
+ * Floats tools library.
+ */
+class Numbers
+{
+	use Nette\StaticClass;
+
+	/** @var float */
+	private static $epsilon = 1e-9;
+
+
+	public static function isZero(float $value): bool
+	{
+		return abs($value) < self::$epsilon;
+	}
+
+
+	public static function isInteger(float $value): bool
+	{
+		return abs(floor($value) - $value) < self::$epsilon;
+	}
+
+
+	public static function divide(float $dividend, float $divisor): float
+	{
+		if (self::isZero($divisor)) {
+			throw new DivisionByZeroException();
+		}
+
+		return $dividend / $divisor;
+	}
+
+
+	public static function percentage(float $total, float $part): float
+	{
+		if (self::isZero($total)) {
+			if (self::isZero($part)) {
+				return 0.0;
+			}
+			throw new DivisionByZeroException('Trying to get a percentage of the total of 0.');
+		}
+		return ($part / $total) * 100;
+	}
+
+
+	public static function compare(float $a, float $b): int
+	{
+		if (abs($a - $b) < self::$epsilon) {
```

Comment:
It's not that simple https://github.com/nette/tester/pull/68

Reply:
Upgraded. Now it works too with INF and NAN.